keras_nlp/models/xlm_roberta/xlm_roberta_tokenizer.py

```diff
@@ -123,6 +123,11 @@ def id_to_token(self, id):
 
     def token_to_id(self, token):
         """Convert a string token to an integer id."""
+
+        # OOV token
+        if token not in self.get_vocabulary():
```

Comment:
Does sentencepiece not handle the oov token in this case? If we can just remap the output that would be better.

This would require loading the full vocab list into a python list and then iterating over it in its entirety. That's probably quite slow!